contact_updater/views.py

```diff
@@ -0,0 +1,111 @@
+import json
+import time
+
+from contact_updater.forms import AgencyData
+
+from django.forms.formsets import formset_factory
+from django.shortcuts import render
+from django.http import HttpResponse
+
+from foia_hub.api import AgencyResource, OfficeResource
+
+
+SITE = "https://foia.18f.us/api/"
```

Comment:
Doesn't look like this is ever used? I'd strong recommend against hard coding this string -- consider using the user's request url if you really need the whole domain.